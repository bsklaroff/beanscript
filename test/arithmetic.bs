a = 2 - 5 * 3 + 4 / 4
b = -a * -a
print(a)
print(b)
print(1 % 2)
print(4 % 2)
print(11 % 3)
print(-11 % 3)
print(-11 % -3)
print(11 % -3)
