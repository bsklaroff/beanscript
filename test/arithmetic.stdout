-12
144
1
0
2
-2
-2
2
